### PR TITLE
Add pointers and sidecar information in the README

### PR DESCRIPTION
**Pull request by humblec**
Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,8 +6,8 @@ Card](https://goreportcard.com/badge/github.com/ceph/ceph-csi)](https://goreport
 Status](https://travis-ci.org/ceph/ceph-csi.svg?branch=master)](https://travis-ci.org/ceph/ceph-csi)
 
 This repo contains [Container Storage Interface(CSI)]
-(<https://github.com/container-storage-interface/>) driver, provisioner,
-and attacher for Ceph RBD and CephFS.
+(<https://github.com/container-storage-interface/>) Ceph CSI driver for RBD, CephFS and kubernetes sidecar deployment
+yamls of provisioner, attacher and snapshotter for supporting CSI functionalities.
```
Comment (Madhu-1) [inline README.md]: add node driver register

File: README.md [hunk 2]
```diff
@@ -17,8 +17,7 @@ attaching them to workloads.
 
 Independent CSI plugins are provided to support RBD and CephFS backed volumes,
 
-- For details about configuration and deployment of RBD and CephFS CSI plugins,
-  see documentation in `docs/`.
+- For details about configuration and deployment of RBD plugin, please refer [this doc](https://github.com/ceph/ceph-csi/blob/master/docs/deploy-rbd.md) and for CephFS plugin please refer [this](https://github.com/ceph/ceph-csi/blob/master/docs/deploy-cephfs.md).
```
Comment (Madhu-1) [inline README.md]: canm we change `this doc` to `deploy-rbd` and  `this` to `deploy-cephfs` or something else

Comment (humblec) [reply README.md]: sure.

File: README.md (hunk 1 above)
Comment (humblec) [reply README.md]: yep.

**Comment by humblec** [comment]
@Madhu-1 finally the md linter allows this to pass :)

**Pull request merged**